### PR TITLE
fix: build problems in 0.6.3

### PR DESCRIPTION
**Pull request by patzick**
# Scope of work

Fixes:
* problem with leaflet causing build problems
image

* problem with not using exact version of shared package, to avoid problems in future
image

* deleted check for *.scss stylelint, as it has some working issue, showed error without pointing that. @pspaczek I think it should be fixed with next release separately
image

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed

### CODE REVIEW

File: packages/vue/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "@glidejs/glide": "^3.3.0",
-    "@storefront-ui/shared": "^0.6.3",
```
Comment (pspaczek) [inline packages/vue/package.json]: @patzick is changed by lerna, can you remove this change

File: packages/vue/src/components/organisms/SfStoreLocator/SfStoreLocator.vue [hunk 2]
```diff
@@ -200,6 +200,7 @@ export default {
     }
   },
   mounted() {
+    import("leaflet/dist/leaflet.css");
```
Comment (pspaczek) [inline packages/vue/src/components/organisms/SfStoreLocator/SfStoreLocator.vue]: @aniamusial @patzick it's some special reason to import css on mounted? Can we move it  to line 65?

File: packages/vue/package.json (hunk 1 above)
Comment (patzick) [reply packages/vue/package.json]: No, Lerna only bumps the number, doesn't interfere with scope. This is the reason why going back to the previous version doesn't work and prevents this kind of blockers for the future. So this line is a crucial part of this fix.

File: packages/vue/src/components/organisms/SfStoreLocator/SfStoreLocator.vue (hunk 2 above)
Comment (patzick) [reply packages/vue/src/components/organisms/SfStoreLocator/SfStoreLocator.vue]: For me doesn't matter, copied from Ania PR :) we need to fix the issue with building so from my perspective better to load this CSS too late for a moment than not to fix the build breaking bug.

Comment (patzick) [reply packages/vue/src/components/organisms/SfStoreLocator/SfStoreLocator.vue]: I believe line 65 is a fair point; I didn't want to interfere with the solution from other PR. Ania - is there a reason why not to do this? :)

File: packages/vue/package.json (hunk 1 above)
Comment (pspaczek) [reply packages/vue/package.json]: right, thanks :)

**Pull request merged**